### PR TITLE
HL-16:  Application lists, docker benefit frontend

### PR DESCRIPTION
**Pull request by melniiv**

### CODE REVIEW

File: docker-compose.benefit.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+version: "3.8"
```
Comment (nenonja) [inline docker-compose.benefit.yml:1]: This could be moved inside `docker-compose.helsinki-benefit.yml`

**Review by nenonja — COMMENTED** [review]
Some comments. Also noting:
- Styles does not seem to work in mobile view. But At least kesaseteli does not have requirements yet for working in mobile view yet so perhaps benefit has neither.
- small glitch with paddings?
Screenshot 2021-06-02 at 2 17 16 PM

**Comment by nenonja** [comment]
workflow does not find dockerfile: https://github.com/City-of-Helsinki/yjdh/pull/57/checks?check_run_id=2722374328

File: docker-compose.benefit.yml (hunk 1 above)
Comment (melniiv) [reply docker-compose.benefit.yml:1]: we will have 1 file, not 2. we will merge into one when backend team makes it work.

File: frontend/benefit/applicant/src/components/applications/styled.ts [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+  margin-bottom: ${(props) => (props.theme as Theme).spacing.xs2};
+`;
+
+const StyledItemContent = styled.div`
+  display: flex;
+`;
+
+const StyledAvatar = styled.div<AvatarProps>`
+  ${(props) =>
+    props.status === APPLICATION_STATUSES.DRAFT &&
```
[the remark this answers is not in the document]
Comment (melniiv) [reply frontend/benefit/applicant/src/components/applications/styled.ts:39]: yes, good idea

**Comment by melniiv** [comment]
> Some comments. Also noting:
> 
> * Styles does not seem to work in mobile view. But At least kesaseteli does not have requirements yet for working in mobile view yet so perhaps benefit has neither.
> * small glitch with paddings?
> 
> Screenshot 2021-06-02 at 2 17 16 PM

yea, its a "feature", i didnt use grid but flex. lets see if i can fix that

**Review by nenonja — APPROVED** [review]

**Pull request merged**